proposals/1219-storing-megolm-keys-serverside.md

```diff
@@ -177,6 +177,42 @@ Error codes:
 
 - `M_NOT_FOUND`: No backup version has been created.
 
+##### `PUT /room_keys/version/{version}`
+
+Update information about the given version, or the current version if `{version}`
```

Comment:
Having the `{version}` optional seems like it makes it a bit too easy for a client to accidentally overwrite a new backup version with an old backup version.  I think we should just make the `version` required.